app/templates/partials/summary/answer-summary.html

```diff
@@ -31,4 +33,8 @@ <h2 class="summary__title neptune">{{ content.summary.label }}</h2>
     {%- endfor -%}
   </div>
   {%- endfor -%}
+  <a class="page__previous page__previous--bottom u-mb-l u-dib"
+     {{helpers.track('click', 'Navigation', 'Add Person link click', bottom)}}
+  href="{{add_person_location}}" id="add-person-link">{{ _("Add ") }} {{ content.summary.label }}
```

Comment:
Since it looks odd as `Add Household members`, can do we one of the below?
`{{ _("Add ") }} {{ content.summary.label|lower }}` -> `Add household members`
or
`{{ _("Add ") }} {{ content.summary.label|title }}` -> `Add Household Members`

Reply:
Also the space after the “Add “ should be in the template and not part of the translated string